src/lib/common/macroSubstitute.cpp

```diff
@@ -160,7 +160,7 @@ bool macroSubstitute(std::string* to, const std::string& from, const Entity& en)
 
     if (macroEnd == std::string::npos)
     {
-      LM_W(("Runtime Error (macro end not found, syntax error, aborting substitution)"));
+      LM_W(("macro end not found, syntax error, aborting substitution"));
```

Comment:
This is not actually due to Runtime Error (i.e. internal error in CB) but a miss-provisioned subscription, eg using:

```
      "headers": {  
      	"X-EntityType": "${type"
      }
```

Thus WARN is ok for the level and "Runtime Error" has to be removed from the trace message.

NTC (informative)